src/ARView.js

```diff
@@ -370,21 +370,8 @@ class ARView {
 
     this.gl.viewport(0, 0, gl.viewportWidth, gl.viewportHeight);
     this.videoRenderer.render();
-    this._resetGLState();
-  }
-
-  /**
-   * Resets the GL state in the THREE.WebGLRenderer.
-   */
-  _resetGLState() {
-    if (typeof this.renderer.resetGLState === 'function') {
-      // If using three.js <= r86
-      this.renderer.resetGLState();
-    } else {
-      // If using three.js >= r87
-      this.renderer.state.reset();
-    }
   }
 }
 
+THREE.ARView = ARView;
```

Comment:
is this supposed to be changed? I might have a merge issue

Reply:
No; we're no longer placing the classes on global THREE here -- if using the script form, it'll attach it (in src/index.js), and if in module form it'll export them from itself via `import { ARView } from 'three.ar.js'`

Reply:
And it fails linting when using global THREE, as expected, so we don't accidentally break the module usage which does not necessarily have THREE globals: https://travis-ci.org/google-ar/three.ar.js/builds/286267888?utm_source=github_status&utm_medium=notification